setup.py

```diff
@@ -34,15 +37,19 @@ def read_file(fname):
         'License :: OSI Approved :: MIT License',
         'Operating System :: OS Independent',
         'Programming Language :: Python',
-        'Topic :: Internet :: WWW/HTTP',
-        'Topic :: Software Development :: Libraries :: Python Modules',
         'Programming Language :: Python :: 3',
         'Programming Language :: Python :: 3.6',
         'Programming Language :: Python :: 3.7',
         'Programming Language :: Python :: 3.8',
         'Programming Language :: Python :: 3.9',
+        'Topic :: Internet :: WWW/HTTP',
+        'Topic :: Software Development :: Libraries :: Python Modules',
     ],
+    python_requires='>=3.6',
     platforms=['any'],
     url='https://github.com/jazzband/django-analytical',
     download_url='https://github.com/jazzband/django-analytical/archive/master.zip',
+    project_urls={
+        'Documentation': 'https://django-analytical.readthedocs.io/',
+    },
```

Comment:
The README has the link to the documentation a bit hidden in the text. So, I thought a prominent link on PyPI's sidebar wouldn't hurt. :grimacing:

Reply:
> Do you think it makes sense (as in "is important") to include those files with the source distribution?

Well, as long as the MANIFEST.in is there at all, it should probably err on the side of consistency in its inclusion, rather than not.

I think the better next step would be to enable `setuptools-scm` for the project, so that we can get rid of MANIFEST.in entirely, and have this managed automatically.

Reply:
Fine with me.